cmd/sync/sync.go

```diff
@@ -158,7 +158,7 @@ func (app *SyncApp) Start(cmd *cobra.Command, args []string) {
 	}
 
 	lg.Info("%v verified layers %v", app.BaseApp.Config.P2P.NodeID, app.sync.ValidatedLayer())
-	lg.Info("sync done")
+	lg.With().EventInfo("sync done")
```

Comment:
this also

Reply:
we need it to be a json since it is an event